unit-tests/py/rspy/test.py

```diff
@@ -243,9 +243,9 @@ def check_exception(exception, expected_type, expected_msg = None, abort_if_fail
     """
     failed = False
     if type(exception) != expected_type:
-        failed = [ "Raised exception was of type", type(exception), "and not of type", expected_type, "as expected" ]
+        failed = [ "    raised exception was of type", type(exception), "\n    but expected type", expected_type ]
     if expected_msg and str(exception) != expected_msg:
-        failed = [ "Exception had message:", str(exception), "\nBut we expected:", expected_msg ]
+        failed = [ "    exception message:", str(exception), "\n    but we expected:", expected_msg ]
```

Comment:
If both the type and the message are different we will only get the message. I think it should be the other way around

Reply:
Fixed, good catch